libbeat/tests/system/test_migration.py

```diff
@@ -0,0 +1,69 @@
+from base import BaseTest
+from nose.plugins.attrib import attr
+from elasticsearch import Elasticsearch, TransportError
+
+import logging
+import os
+import shutil
+import unittest
+
+
+INTEGRATION_TESTS = os.environ.get('INTEGRATION_TESTS', False)
+
+
+class TestCommands(BaseTest):
+    """
+    Test beat subcommands
+    """
+
+    def setUp(self):
+        super(BaseTest, self).setUp()
+        shutil.copy(self.beat_path + "/_meta/config.yml",
+                    os.path.join(self.working_dir, "libbeat.yml"))
+
+    def test_migration_default(self):
+        """
+        Tests that if no migration flag is set, no alias exists. By default migratin is off.
+        """
+
+        exit_code = self.run_beat(
+            extra_args=[
+                "export", "template",
+                "-E", "setup.template.fields=" + os.path.join(self.working_dir, "fields.yml"),
+            ],
+            config="libbeat.yml")
+
+        assert exit_code == 0
+        assert self.log_contains_count('"type": "alias"') == 0
```

Comment:
Should we check for an specific name? I guess we can still have fields of type alias no matter the config for migration. Same for other checks here.

Reply:
yes, will change that. This was a quick win as we didn't have any other aliases.